CHANGELOG.md

```diff
@@ -1,19 +1,27 @@
 # Main (unreleased)
 
-# v0.19.0 (2021-09-29)
-
-This release has breaking changes. Please read [CHANGE] entries carefully and
-consult the
-[upgrade guide](https://github.com/grafana/agent/blob/main/docs/upgrade-guide/_index.md)
-for specific instructions.
-
+- [FEATURE] Operator: The Grafana Agent Operator can now generate a Kubelet
+  service to allow a ServiceMonitor to collect Kubelet and cAdvisor metrics.
+  This requires passing a `--kubelet-service` flag to the Operator in
+  `namespace/name` format (like `kube-system/kubelet`). (@rfratto)
+  
 - [ENHANCEMENT] Updated the following exporters to the newest version available:
```

Comment:
Can we make each of these separate items and give credit to @gaantunes for each?